### ISSUE TITLE
IPhone Safari image not show,when I insert image. 

### ISSUE THREAD

**Issue by jackRoadOnTop** [fraywing/textAngular#1473]
taRegisterTool('selectImage', {
      iconclass: 'fa fa-picture-o',
      action: function() {
        var imageLink = "https://vis-workworld-image.oss-cn-qingdao.aliyuncs.com/undefined/undefined_1482139607387.jpg";
        var thisd = this;
        if (window.cordova) {
          addOnlinePhotoService.show(onSuccessCall);

          function onSuccessCall(result) {
            console.log(result);
            if (result.responseCode == 200) {
              if (device.platform.toLowerCase() === 'android') {
                //json返回格式问题，故用eval
                var retObj = eval('(' + result.response + ')');
                console.log(retObj);
                var embed = '<img src="' + retObj.retmsg + '"><br/>';
                console.log(embed);
                console.log("android");
                thisd.$editor().wrapSelection('insertHTML', embed, true);
              } else if (device.platform.toLowerCase() === 'ios') {
                var retObj = eval('(' + result.response + ')');
                console.log(retObj);
                var embed = '<span style="position: relative;"><img src="' + retObj.retmsg + '"></span>';
                thisd.$editor().wrapSelection('insertHTML', embed, true);
                alert('IOS');
              }
            }
          }
        } else {
          var embed = '<img src="' + imageLink + '"><br/>';
          thisd.$editor().wrapSelection('insertHTML', embed, true);
        }
      }
    })

**Comment by jackRoadOnTop** [comment]
Android device is ok，ios not show.

**Comment by pbassut** [comment]
Is the html inserted though?

**Comment by jackRoadOnTop** [comment]
Yes.Finally,I find https://forum.ionicframework.com/t/is-there-any-rich-text-editor-to-use-in-my-hybrid-app/4437/32

**Comment by jackRoadOnTop** [comment]
Reason found did not find a solution.
